yhteystiedot.html

```diff
@@ -20,6 +20,7 @@ <h2>Contact Information</h2>
             <ul>
                 <!-- Contact information content -->
             </ul>
+            <p>Lisätään tähän tekstiä tehtävän suorittamista varten jotta saadaan commit tehtyä Azure Boardsin itemia 33 varten.</p>
```

Comment:
Onko tämä line oikein?

Reply:
Tämä on juurikin oikein, hienosti tehty!

Reply:
Just näin. Tällä eteenpäin.